tools/cbmc/proofs/DNS/DNShandlePacket_harness.c

```diff
@@ -0,0 +1,28 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_DNS.h"
+#include "FreeRTOS_IP_Private.h"
+
+// Here we are assuming that prvParseDNSReply is implemented correctly.
```

Comment:
But we've justified that assumption via our separate proof here https://github.com/aws/amazon-freertos/tree/master/tools/cbmc/proofs/ParseDNSReply correct?

Reply:
Yes, prvParseDNSReply is proved already.

Reply:
I have edited the comment to point to the other proof.